packages/graz/types/global.d.ts

```diff
@@ -1,16 +1,20 @@
 type KeplrWindow = import("@keplr-wallet/types").Window;
 type VectisWindow = import("@vectis/extension-client").VectisWindow;
+import Station from "@terra-money/station-connector";
 
-interface Window extends KeplrWindow, VectisWindow {
-  leap: KeplrWindow["keplr"];
-  cosmostation: {
-    cosmos: {
-      on: (type: string, listener: EventListenerOrEventListenerObject) => void;
-      off: (type: string, listener: EventListenerOrEventListenerObject) => void;
+declare global {
```

Comment:
any reason to add this @apollo-sturdy? what do you think @grikomsn

Reply:
Without this the other wallet integrations (vectis, leap) broke. I suspect since importing `@terra-money/station-connector` does the same global declaration and overwrites ours. You can confirm if you remove it that you get type errors in `actions/wallet/vectis.ts`.